sematic/db/models/resolution.py

```diff
@@ -151,7 +152,7 @@ class ResolutionKind(Enum):
     KUBERNETES = "KUBERNETES"  # for detached mode
 
 
-class Resolution(HasUserMixin, Base, JSONEncodableMixin):
+class Resolution(HasUserMixin, HasOrganizationMixin, Base, JSONEncodableMixin):
```

Comment:
I see we already have mixins before `Base` 😆 . I leave it to your discretion, but I'd prefer them after `Base`.

Reply:
Done.